### PR TITLE
fixes the loris user id to 1002

### PR DESCRIPTION
**Pull request by lsh-0**
* same as prod and end2end and container image

### CODE REVIEW

File: salt/iiif/loris.sls [hunk 1]
```diff
@@ -75,12 +75,12 @@ loris-dependencies:
             - loris-repository
             - pkg: loris-dependencies
 
-# lsh 2019-03-19: why a special 'loris' user and not www-data?
```
Comment (giorgiosironi) [inline salt/iiif/loris.sls]: I think it was ported over from the documentation of the project

Comment (lsh-0) [reply salt/iiif/loris.sls]: yeah, it was just me leaving open questions to my future-self really. Saves me asking them again later ;)

**Comment by lsh-0** [comment]
> More reproducibility

That and I need to sync what we have with the user inside the container.

**Comment by lsh-0** [comment]
nuts. this breaks vagrant. 

vagrant user is 1000
ubuntu user is 1001
elife user is 1002
loris user (1002) creation fails

**Pull request merged**